docs/docs/data-source/2-data-source-options.md

```diff
@@ -28,12 +28,6 @@ Different RDBMS-es have their own specific options.
     Example: `subscribers: [PostSubscriber, AppSubscriber, "subscriber/*.js", "modules/**/subscriber/*.js"]`.
     Learn more about [Subscribers](../advanced-topics/4-listeners-and-subscribers.md).
 
--   `migrations` - Migrations to be loaded and used for this data source.
-    It accepts both migration classes and directories from which to load.
-    Directories support glob patterns.
-    Example: `migrations: [FirstMigration, SecondMigration, "migration/*.js", "modules/**/migration/*.js"]`.
-    Learn more about [Migrations](../advanced-topics/1-migrations.md).
```

Comment:
we should keep this info in these docs, even if we duplicate some of it to the migration docs

Reply:
Why would we? Similarlity to CLI docs, I think this page should only be _an index_ linking to more detailed pages...